bentoml/yatai/python_api.py

```diff
@@ -118,16 +119,26 @@ def upload_bento_service(bento_service, base_path=None, version=None):
                     "is {}".format(http_response.status_code, http_response.text)
                 )
 
+            update_bento_upload_progress(yatai, bento_service)
+
+            request = GetBentoRequest(
+                bento_name=bento_service.name, bento_version=bento_service.version
+            )
+
+            response = yatai.GetBento(request)
+            if response.status.status_code != Status.OK:
+                raise BentoMLException(
+                    "Error getting Bento with status code {} ".format(response.status)
+                )
+
             logger.info(
                 "Successfully saved Bento '%s:%s' to S3: %s",
                 bento_service.name,
                 bento_service.version,
-                response.uri.uri,
+                response.bento.uri.uri,
```

Comment:
actually I'd prefer to just print the original s3 address here, since users can always use that as a reference to verify it with aws cli or aws web console, and it's also easier to integrate with other tools.

One thing we need to change soon is from ```load(archive_path)``` to ```load(name, version)```, I think once we did that, we won't need the `saved_path` in your example to be the downloadable URL, so it remains the original s3 url.

One thing I do wish we can do now, is directory loading from an address like "s3://my_repo/my_svc/version_id/my_svc.tar.gz" if there user's machine already has aws configured. that way the loading can still work, it's just not via a presigned URL like we do with our yatai client.

Reply:
yep, but the original s3 address (`response.uri.uri`) is just a bucket domain name (like this `https://test-bento.s3-ap-northeast-1.amazonaws.com`) without the key.

+1 for `load(name, version)`

an address like "s3://my_repo/my_svc/version_id/my_svc.tar.gz" does look much simpler, but this file `python_api.py`  may be on a host without S3 keys (because the keys are configured at yatai server) and I assume the bucket is more frequently configured as a private bucket than a public bucket, then this will result in an Access Denied URL.